README.md

```diff
@@ -55,6 +55,19 @@ grype path/to/image.tar
 grype dir:path/to/dir
 ```
 
+Use [Syft](https://github.com/anchore/syft) SBOMs for even faster vulnerability scanning in Grype:
+
+```
+# Just need to generate the SBOM once
```

Comment:
This looks great but it would be nice to break out the comment out of the code block

Reply:
Discussed in Slack — this is consistent with how we approach our examples today, but we should heavily consider refreshing out READMEs and expanding into a docs site